cmd/gateway-main.go

```diff
@@ -136,11 +137,17 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 	// Handle common env vars.
 	handleCommonEnvVars()
 
+	// Get port to listen on from gateway address
+	_, gatewayPort, pErr := net.SplitHostPort(gatewayAddr)
```

Comment:
I chose to use `net.SplitHostPort` over `mustSplitHostPort` because the former allows me to add context to the fatal error message we will be displaying to the user.

## Using `net.SplitHostPort`
```
ERROR Unable to start gateway: address 9001: missing port in address.
```
## Using `mustSplitHostPort`
```
ERROR Unable to split host port 9001: address 9001: missing port in address.
```